build/pool-manager/Dockerfile

```diff
@@ -30,5 +30,5 @@ LABEL org.label-schema.build-date=$DBUILD_DATE
 LABEL org.label-schema.vcs-url=$DBUILD_REPO_URL
 LABEL org.label-schema.url=$DBUILD_SITE_URL
 
-ENTRYPOINT entrypoint.sh
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

Comment:
@mittachaitu Does it mean that PID 1 will be `/usr/local/bin/entrypoint.sh` instead of `/bin/sh -c /usr/local/bin/entrypoint.sh`

Reply:
Yes @akhilerm